src/command-line/index.js

```diff
@@ -7,7 +7,7 @@ var fsextra = require("fs-extra");
 var path = require("path");
 var Helper = require("../helper");
 
-program.version(pkg.version, "-v, --version");
+program.version(Helper.getVersion(), "-v, --version");
```

Comment:
I remember getting the same feeling when implementing the UI version. This is what should be displayed:

1. If this is a git repository, git is available and git command is available, display git commit.
2. Otherwise, display npm version

As an amendment to (1.), if the commit hash has an exact match for an npm version (i.e. a tag), we may want to display npm version instead. Though it might be a bit confusing as, essentially, instance **is** running from source and not from a build... so I'd say, if git repo + git available, go for git commit.

However, that npm version + commit hash (like in your picture above) is a no-no for me. As a user, what does that mean, am I running the pre.6 version or a specific commit? Is that commit hash related to that version? If I notice a bug, does that mean pre.6 have to be incriminated?
I'm in favor of simplicity here: an npm version, or a commit hash, but not both.